src/convertToObject.js

```diff
@@ -7,6 +7,18 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const elements = sourceString.split(';');
+  const result = {};
+
+  elements.forEach(function (element) {
+    const [name, value] = element.split(':');
+
+    if (name && value) {
+      result[name.trim()] = value.trim();
+    }
+  });
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. You are already using the forEach method, which is appropriate for this task.